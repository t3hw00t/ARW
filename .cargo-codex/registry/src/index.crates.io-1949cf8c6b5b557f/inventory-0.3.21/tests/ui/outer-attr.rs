#![feature(used_with_arg)]

struct Thing;

inventory::collect!(Thing);

inventory::submit! {
    #[used(linker)]
    Thing
}

fn main() {}
