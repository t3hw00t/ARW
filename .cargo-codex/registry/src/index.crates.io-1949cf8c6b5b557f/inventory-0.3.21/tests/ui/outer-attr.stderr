error: `#[used]` attribute cannot be used on expressions
 --> tests/ui/outer-attr.rs:8:5
  |
8 |     #[used(linker)]
  |     ^^^^^^^^^^^^^^^
  |
  = help: `#[used]` can only be applied to statics
